torch2trt/converters/repeat.py

```diff
@@ -0,0 +1,28 @@
+import tensorrt as trt 
+from torch2trt.torch2trt import *
+from torch2trt.module_test import add_module_test
+
+@tensorrt_converter('torch.Tensor.repeat', enabled=trt_version() >= '6.0')
+def convert_repeat(ctx):
+    inputs = get_arg(ctx, 'input', pos=0, default=None) 
+    output = ctx.method_return
+    output_list = output.detach().cpu().numpy()
+    op_shape = list(output.size())
+    layer = ctx.network.add_constant(shape=trt.Dims(op_shape),weights=trt.Weights(output_list))
```

Comment:
Looks like this is adding a constant, which may give some false-positive conversions (where a dynamic input is added as a constant).  

Have you looked into using different TensorRT method to supporting non-constant inputs?

Reply:
I tried `add_concatenation` method. However, it only concatenates a list of input tensors and I can't explicitly say on which dimension to concatenate. So it will fail on the following use case:

```
x = torch.tensor([1, 2, 3])
x.repeat(4, 2)
```

I cant think of any other method.

Reply:
Hmm, I do see that it doesn't allow axis specification for ``add_concatenation``

https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Graph/Network.html#tensorrt.INetworkDefinition.add_concatenation

I haven't personally tried, but it looks like there is an ``axis`` variable for the returned layer which controls this.

https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Graph/Layers.html#iconcatenationlayer

Have you tried setting this variable after adding the layer?

```python
layer = network.add_concatenation(...)
layer.axis = ...
```